### PR TITLE
243 fix the minor issues on riskmap

### PR DESCRIPTION
**Pull request by Ssravani272**

### CODE REVIEW

File: src/components/disaster-map/map-layers.js [hunk 1]
```diff
@@ -851,6 +851,7 @@ export class MapLayers {
             self.getData("reports/" + reportId)
                 .then(data => {
                     self.reports.addData(data);
+                    console.log(data,'datareport');
```
Comment (Deepthi-Chand) [inline src/components/disaster-map/map-layers.js:854]: Remove console logs

File: src/components/disaster-map/map-utility.js [hunk 2]
```diff
@@ -149,7 +151,8 @@ export class MapUtility {
     }
 
     noReportNotification(cityName, reportId) {
-        if (reportId && cityName) {
+        console.log(this.statsReports);
```
Comment (Deepthi-Chand) [inline src/components/disaster-map/map-utility.js:154]: remove console logs

**Review by Deepthi-Chand — CHANGES_REQUESTED** [review]

File: src/components/disaster-map/map-layers.js (hunk 1 above)
Comment (Ssravani272) [reply src/components/disaster-map/map-layers.js]: Yeah

File: src/components/disaster-map/map-utility.js (hunk 2 above)
Comment (Ssravani272) [reply src/components/disaster-map/map-utility.js]: Yeah

**Review by Deepthi-Chand — APPROVED** [review]

**Pull request merged**